### DISCUSSION TITLE
dd not showing up when hitting controller

### DISCUSSION THREAD

**Discussion by Ifrit123** [symfony/symfony#48745] [Q&A]
I've just started learning how to use symfony and I've got xampp installed on my windows 10.
The issue I'm having is that when I go to movies.test/movies I'm getting the `The requested URL was not found on this server.` error

My code on the controller
```
<?php

namespace App\Controller;

use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
use Symfony\Component\HttpFoundation\Response;
use Symfony\Component\Routing\Annotation\Route;

class MoviesController extends AbstractController
{
      #[Route('/movies', name: 'movies')]
    public function index(): Response
    {
        dd('im here');
      }
}
```

**Comment by yarscript** [comment]
@Ifrit123 Check if mod_rewrite is enabled at your apache web server
